test/default_mounts.bats

```diff
@@ -2,31 +2,26 @@
 
 load helpers
 
-IMAGE="quay.io/crio/redis:alpine"
 
-function teardown() {
-	cleanup_test
-}
 
 @test "bind secrets mounts to container" {
     start_crio
     run crictl runp "$TESTDATA"/sandbox_config.json
     echo "$output"
     [ "$status" -eq 0 ]
     pod_id="$output"
-    run crictl pull "$IMAGE"
-    [ "$status" -eq 0 ]
     run crictl create "$pod_id" "$TESTDATA"/container_redis.json "$TESTDATA"/sandbox_config.json
     echo "$output"
     [ "$status" -eq 0 ]
     ctr_id="$output"
-    run crictl exec --sync "$ctr_id" cat /proc/mounts
+    run crictl exec --sync "$ctr_id" ls /run/secrets
```

Comment:
I'd rather add a new test than changing an old one just to verify a new condition. Please add a new one for all these cases

Reply:
Fixed.